docs/source/federated-types/composition.mdx

```diff
@@ -184,23 +184,23 @@ If the gateway sent this query to Subgraph A, it would return an error. And with
 
 Composition recognizes this potential issue, and it fails. The hypothetical supergraph schema above would never actually be generated.
 
-`Position.z` is an example of a field in a shared type that is _not_ always resolvable. So now, how do we make sure that such a field _is_ always resolvable?
+`Position.z` is an example of a shared field that is _not_ always resolvable. So now, how do we make sure that such a field _is_ always resolvable?
 
 ### Solutions for unresolvable fields
 
 There are multiple solutions for making sure that a field of a shared type is always resolvable. Choose a solution based on your use case:
 
 #### Define the field in every subgraph that defines the type.
 
-If every subgraph that defines a type _could_ resolve every field of that type without introducing complexity, a straightforward solution is to define and resolve all fields in all of those subgraphs:
+If every subgraph that defines a type _could_ resolve every shared field of that type without introducing complexity, a straightforward solution is to define and resolve all fields in all of those subgraphs:
```

Comment:
I might stick with just "field" here as well, just because sharing all of the fields is part of the ultimate solution shown in the syntax